src/main/java/com/redhat/exhort/tools/Operations.java

```diff
@@ -89,13 +90,30 @@ public static void runProcess(final String... cmdList) {
 
     // verify the command was executed successfully or throw a runtime exception
     if (exitCode != 0) {
-      throw new RuntimeException(
-        String.format(
-          "failed to execute '%s', exit-code %d",
-          String.join(" ", cmdList),
-          exitCode
-        )
-      );
+      String errMsg = new BufferedReader(new InputStreamReader(process.getErrorStream()))
+        .lines().collect(Collectors.joining("\n"));
```

Comment:
@xieshenzh Can you please replace "\n" with System.lineSeparator()?
So it will be portable, and will work  as expected and will look the same also on windows environment.